torchani/models.py

```diff
@@ -76,6 +79,14 @@ def get_resource(file_path):
 
         info_file = get_resource(info_file_path)
 
+        # download resources if not already exist
+        if not os.path.isfile(info_file):
+            print('Downloading ANI model parameters ...')
+            resources_url = "https://www.dropbox.com/sh/otrzul6yuye8uzs/AABuaihE22vtaB_rdrI0r6TUa?dl=1"
```

Comment:
Instead of hosting a dropbox, can we ask @isayev to create release for `ASE_ANI` and download the zip from the release?

Reply:
Yeah, that's also a good idea, but the release will encapsulate other directories in `ASE_ANI` into the zip file resulting in a larger directory to download/request

Reply:
I believe the other files are much smaller than the models? But this shouldn't be a blocker. We could merge this PR with this link, and change the URL later if we want.

Reply:
Avoiding Dropbox links is mostly for security concerns. If we use dropbox links, does this mean that we can put malware on that link, and users will download and run it without being notified? GitHub release has much better visibility.

Reply:
@zasdfgbnm sure I could do that. Alternatively, we could create a separate GitHub repo just for models and make releases there.